AUTHORS.rst

```diff
@@ -2,20 +2,21 @@
 Credits
 =======
 
-Development Lead
-----------------
-
+Development Leads
```

Comment:
I don't like the `Development Lead` bucket at all. I kept it from the previous author. I would be more than happy if we just have an `Active Contributors`  and `Past Contributors`

Reply:
Done!